scripts/zones/Cape_Teriggan/mobs/Axesarion_the_Wanderer.lua

```diff
@@ -19,8 +19,12 @@ entity.onMobDisengage = function(mob)
 end
 
 entity.onMobDeath = function(mob, player, optParams)
-    if optParams.isKiller then
-        GetNPCByID(ID.npc.CERMET_HEADSTONE):setLocalVar('cooldown', GetSystemTime() + 900)
+    local headstone = GetNPCByID(ID.npc.CERMET_HEADSTONE)
+    if
+        headstone and
+        (optParams.isKiller or optParams.noKiller)
+    then
+        headstone:setLocalVar('cooldown', GetSystemTime() + 900)
```

Comment:
You get the idea by now. Update them all.

onMobDeath runs once per entity in the mob enmity list. We do not want to fetch an entity (headstone in this case) 6+ times only to use it once. We want to use it just 1 time, so we fetch it 1 time only.